examples/word_prediction/readme_examples.py

```diff
@@ -0,0 +1,43 @@
+from happytransformer import HappyWordPrediction
+
+
+def example_1_0():
+    happy_wp_distilbert = HappyWordPrediction()  # default
+    happy_wp_albert = HappyWordPrediction("ALBERT", "albert-base-v2")
+    happy_wp_bert = HappyWordPrediction("BERT", "bert-base-uncased")
+
+
+def example_1_1():
+    happy_wp = HappyWordPrediction()  # default uses distilbert-base-uncased
+    result = happy_wp.predict_mask("I think therefore I [MASK]")
+    print(type(result))  # <class 'list'>
+    print(result)  # [WordPredictionResult(token_str='am', score=0.10172799974679947)]
+    print(type(result[0]))  # <class 'list'>
```

Comment:
by isnspection, `type(result[0])` should be `WordPredictionResult`

Reply:
Nice catch, I was copying and pasting stuff and forgot to change the comment.